extension.json

```diff
@@ -1,59 +1,64 @@
 {
 	"name": "Page Approvals",
-
 	"version": "1.0.0",
-
 	"author": [
 		"[https://Professional.Wiki/ Professional.Wiki]"
 	],
-
 	"license-name": "BSD-3-Clause",
-
 	"url": "https://github.com/ProfessionalWiki/PageApprovals",
-
 	"descriptionmsg": "pageapprovals-description",
-
 	"requires": {
 		"MediaWiki": ">= 1.39.0"
 	},
-
 	"MessagesDirs": {
 		"PageApprovals": [
 			"i18n"
 		]
 	},
-
 	"AutoloadNamespaces": {
 		"ProfessionalWiki\\PageApprovals\\": "src/",
+		"ProfessionalWiki\\PageApprovals\\HookHandler\\": "src/HookHandler/",
```

Comment:
Hooks go into the `EntryPoints` directory. Example: https://github.com/ProfessionalWiki/PageApprovals/pull/11/files#diff-747c2e96de0f81fec35a95634faab42f883963f6e596f8a5c8f1254326649db9

Reply:
I will do that now but I was confused to see why we had separate folder for HookHander in ProAdmin.